src/main/java/io/netty/incubator/codec/quic/QuicSslContextBuilder.java

```diff
@@ -178,6 +179,14 @@ public QuicSslContextBuilder earlyData(boolean enabled) {
         return this;
     }
 
+    /**
+     * Enable / disable keylog
```

Comment:
Do we also want to document which logger it uses `io.netty.incubator.codec.quic.BoringSSLKeylogCallback` ?